com.hunterwittenborn.Celeste.yml

```diff
@@ -0,0 +1,69 @@
+app-id: com.hunterwittenborn.Celeste
+runtime: org.gnome.Platform
+runtime-version: "43"
+sdk: org.gnome.Sdk
+command: celeste
+finish-args:
+  # We need to display on Wayland/X11.
+  - '--socket=wayland'
+  - '--socket=fallback-x11'
+  # Enable OpenGL rendering.
+  - '--device=dri'
+  # Users may want to sync files from anywhere on their system.
+  - '--filesystem=host'
+  # We need network access in order to sync files to remotes.
+  - '--share=network'
+  # Used for X11 performance as described at https://docs.flatpak.org/en/latest/sandbox-permissions-reference.html#f1.
+  - '--share=ipc'
+  # I haven't been able to find out why, but we have to have access to the full
+  # DBus session or the tray stops running due to not being able to send
+  # messages to the main application (the tray app was designed to quit when it
+  # can't any messages to the main application). Neither the Debian package nor
+  # the Snap are affected by the issue, so I'm not quite sure what's up with the
+  # Flatpak.
+  - '--socket=session-bus'
+  # Extra libraries that we've bundled with the Flatpak in order to run. '/usr/lib/x86_64-linux-gnu' is exposed by default, but if we specify one we have to specify them all.
+  - '--env=LD_LIBRARY_PATH=/usr/lib/x86_64-linux-gnu:/app/lib/x86_64-linux-gnu'
```

Comment:
it was also mentionned you don't need this, these path are looked up by default.

Reply:
If I remove that I get errors at runtime about finding `libappindicator3.so.1`, which is located at `/app/lib/x86_64-linux-gnu/libappindicator3.so.1`. My application bundles the tray icon executable inside of it, which it then writes to a file and executes at runtime - I don't know how related that might be, but my testing is showing it's not working without that line.